test/test_object.py

```diff
@@ -100,7 +101,7 @@ def test_dict_like(self):
       a.method_that_doesnt_exist_on_dict()
 
     with pytest.raises(KeyError):
-      a['xyz']
+      _ = a['xyz']
```

Comment:
is this better than disabling the lint error inline?

Reply:
yeah, i didn't think too hard about it; I can disable the error instead. since it's a one-liner that is pretty clearly trying to raise this error, i figured either would be fine, but if it were more complicated for some reason, i'd agree that disabling is clearer of intent

Reply:
if it's a new pylint error (which i assume is the case) then we will need to inline the disable later, otherwise pylint will say it doesn't understand the issue